pkg/generator/templates/cloud-init-ubuntu.template

```diff
@@ -9,6 +9,18 @@ write_files:
   content: |
     bmV0d29yazoKICBjb25maWc6IGRpc2FibGVkCg==
 {{ end -}}
+{{ if .IsContainerDEnabled -}}
+- path: '/etc/systemd/system/containerd.service.d/11-exec_config.conf'
+  permissions: '0644'
+  encoding: b64
+  content: |
+    W1NlcnZpY2VdCkV4ZWNTdGFydD0KRXhlY1N0YXJ0PS91c3IvYmluL2NvbnRhaW5lcmQgLS1jb25maWc9L2V0Yy9jb250YWluZXJkL2NvbmZpZy50b21s
```

Comment:
```
[Service]
ExecStart=
ExecStart=/usr/bin/containerd --config=/etc/containerd/config.toml
```

What's the reason for the empty `ExecStart`?

Reply:
The only way to override the ExecStart in a drop-in is first set an empty ExecStart (clear it)